### ISSUE TITLE
Make run-server script portable

### ISSUE THREAD

**Issue by abcorrea** [prost-planner/prost#84]
The run-server script is written in Bash syntax. Thus, it does not work on environments using zsh, sh, etc. Ideally, the script should be written in POSIX syntax, instead of Bash.

This should be a trivial issue to solve

**Comment by thomaskeller79** [comment]
@abcorrea can you have a look at the pull request here:

https://github.com/thomaskeller79/prost-dev/pull/3

**Comment by thomaskeller79** [comment]
This issue has been resolved in 23f109e1fcced27b311fa0df1b15180570ec40f8

**Comment by geisserf** [comment]
I get the following error when running the script with --all-benchmarks:

```
Running all benchmarks...
Traceback (most recent call last):
  File "./run-server.py", line 87, in <module>
    subprocess.check_call(["java", "-Xms{}".format(args.initial_memory_alloc),
AttributeError: 'Namespace' object has no attribute 'initial_memory_alloc'
```

**Comment by thomaskeller79** [comment]
@geisserf I implemented your suggestions and made quite a few more changes. Can you review this?

**Comment by geisserf** [comment]
Yes, I will try to do this still today on my end.

**Comment by geisserf** [comment]
Fixed in c1cb39bae02fc573721217e5d9b088e01c784261.
